### DISCUSSION TITLE
BUG: Nightly version 2122091407 & 2122091408

### DISCUSSION THREAD

**Discussion by master3395** [NightscoutFoundation/xDrip#2348] [General]
Hello.

Bug category is missing on "Category" here.

I have a OnePlus 8 Pro Android 12.1
I have been trying to get the latest version of xDrip+ 2122091408.
But every time I try to install it.
It falls back to 2122091407, which I already have installed.

I restarted the phone, to make sure that wasn't an issue.
I have checked that all permissions are given to xdrip+ and everything is given.
I have tried external download and install, still says "finished" at the end, but it returns to 2122091407.

**Comment by Navid200** [comment]
Please post a screenshot of the system status page.
You can blackout any IDs or serial numbers.
But, please leave the code version visible.

**Reply by master3395** [reply]
> Please post a screenshot of the system status page. You can blackout any IDs or serial numbers. But, please leave the code version visible.

![Screenshot_2022-09-21-17-28-04-27_2](https://user-images.githubusercontent.com/4199669/191546609-0cdaa66e-e16d-4ee6-a29c-ef08191a6bc4.jpeg)

**Reply by Navid200** [reply]
That's the last Nightly release you have there.  
Why do you think it has not installed it?

**Reply by master3395** [reply]
> That's the last Nightly release you have there. Why do you think it has not installed it?

This is why I thought there was a new version.
I clicked "check for updates" and got the image below.
![image](https://user-images.githubusercontent.com/4199669/191591413-652c6a30-6eca-48aa-941a-f74e689eccc1.png)

**Reply by Navid200** [reply]
Thanks for reporting this.
Please give us time to investigate.  I will try to recreate this and hopefully identify why it happens.

**Reply by Navid200** [reply]
I can see the same behavior.

**Comment by Navid200** [comment]
As of a few seconds ago, a new Nightly has been released and this problem is now resolved.

Thanks for reporting.
